packages/palette/src/Theme.tsx

```diff
@@ -6,7 +6,7 @@ import { GridThemeProvider as StyledGridThemeProvider } from "styled-bootstrap-g
 import { ThemeContext, ThemeProvider } from "styled-components"
 import { Theme as TTheme, THEME_V2, THEME_V3, ThemeV2, ThemeV3 } from "./themes"
 
-export * from "./themes/v2"
+export * from "@artsy/palette-tokens/dist/themes/v2"
```

Comment:
Wondering if you're going to have to just manually list out the specific exports as these are failing in type-check

Reply:
Type-checking passes locally, but for some reason it's blowing up in CI. I'm not sure why. I even pointed things at canaries.